libs/components/manifest/src/get-public-api.test.ts

```diff
@@ -23,44 +23,4 @@ describe('get-public-api', () => {
 
     expect(result).toEqual(publicApi);
   });
-
-  it('should return the public API for a specific docs ID', async () => {
```

Comment:
I'm going to rework this in a future pull request since the paradym changed a bit.